projects/storefrontlib/src/lib/ui/layout/config/default-layout-config.ts

```diff
@@ -19,6 +19,20 @@ export const defaultLayoutConfig: LayoutConfig = {
         'Section5'
       ]
     },
+    CategoryPageTemplate: {
+      slots: ['Section1', 'Section2', 'Section3']
+    },
+    ProductListPageTemplate: {
+      slots: ['ProductListSlot', 'ProductLeftRefinements']
+    },
+    SearchResultsListPageTemplate: {
+      slots: [
+        'Section2',
```

Comment:
Proviously `SearchResultsListPageTemplate` didn't have `Section2` and `Section4` or maybe I musunderstood something?